doc/news/changes/minor/20171113GiovanniAlzetta

```diff
@@ -0,0 +1,3 @@
+New: GridTools::guess_point_owner with tests(). Given a vector of points uses bounding boxes describing the mesh (see GridTools::compute_mesh_predicate_bounding_box() ) to guess the owners.
```

Comment:
This description is not very clear. May I suggest something like

New: Given a vector of points, the GridTools::guess_point_owner with tests() function uses bounding boxes describing the mesh to guess the processes that own these points.